ground/gcs/src/plugins/welcome/qml/main.qml

```diff
@@ -90,8 +90,8 @@ Rectangle {
 
                 WelcomePageButton {
                     baseIconName: "hitl"
```

Comment:
Yeah, it does: https://github.com/d-ronin/dRonin/blob/next/ground/gcs/src/plugins/welcome/qml/WelcomePageButton.qml#L16